docs/docs/visual-testing-with-storybook.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Visual Testing with Storybook
+---
+
+Knowing your components look as intended in every permutation is not only a great way to test them visually, but also provides "living documentation" for them. This makes it easier for teams to 1) know what components are available to them in a given project and 2) what props those components accept and what all of the states of that component are. As your project grows over time having this information available will be invaluable.
```

Comment:
I would add a quick one line defining what Storybook is

Reply:
Could just copy directly from https://storybook.js.org/basics/introduction/